examples/algorithm_progression.py

```diff
@@ -0,0 +1,172 @@
+import argparse
+import concurrent.futures
+import inspect
+import itertools
+import os
+from pathlib import Path
+
+import algorithms
+import numpy as np
+import yaml
+
+import large_image
+
+VARIABLE_LAYERS = ['z', 'c', 't']
+
+
+argparser = argparse.ArgumentParser(
```

Comment:
It is more common to have all of the argparse stuff into the `if __name__ == '__main__':` conditional, or to make a function that creates the parser and call that from within that conditional.

Reply:
Changed in [8f12e60](https://github.com/girder/large_image/pull/1051/commits/8f12e60fa621de791b9463118c999c13ddb2e88a)